tests/collections/tts/test_waveglow.py

```diff
@@ -54,7 +54,10 @@
 def input_example(sz):
```

Comment:
Can we undo the changes to this file?

Reply:
Why, it would fail to compile then? See my comment above.